client/src/components/overview/Cart.jsx

```diff
@@ -57,19 +57,19 @@ const Cart = ( { stylesList, style, handleStyleSelect, currentProduct } ) => {
       <div className='styles-box'>
         <h1><strong>Style</strong> > <em>{style.name}</em></h1>
         <div className='styles'>
-        {stylesList.map(style => {
-          return <div className='style' key={style.style_id}>
-            <label>
-              <span className='style-caption'>
-                <span>{style.name}</span>
-                </span>
-              <input type="radio" name="check" className='check'/>
-              <img
-              key={style.style_id}
-              pic-id={style.style_id}
-              src={style.photos[0].thumbnail_url}
-              onClick={(e) => handleStyleSelect(e.currentTarget.getAttribute('pic-id'))}/>
-            </label>
+        {stylesList.map( (stylePic, index) => {
+          return <div className='style' key={stylePic.style_id}>
+            <span className='style-caption'>{stylePic.name}</span>
+            <img
+            className={
+              (stylePic === style)
+              ? 'style-selected'
+              : 'style-not-selected'}
+            key={stylePic.style_id}
+            id={index+100}
```

Comment:
why is this index+100?

Reply:
Just a random index value for now. Probably will delete later